QuadradiusR/scripts/board.gd

```diff
@@ -100,3 +117,13 @@ func _torus_putdown(torus: Node):
 		return
 	
 	self.active_torus.current_tile.set_slot(self.active_torus)
+
+
+# # # #
+# ??? #
```

Comment:
???

Reply:
This is a placeholder section, right now I don't know how it should be called. I can just leave the function at the top of the file but it looks kinda worse